app/templates/createclasssuccess.html

```diff
@@ -8,7 +8,6 @@
         <h1>Class was successfully created!</h1>
         <h1>Class code: <span>{{ group.classCode }}</span></h1>
         <a href="{{ url_for('forum', groupID=group.id) }}" class="route-btn">Go To Class</a>
-        <a href="{{ url_for('dashboard') }}" class="route-btn">Create A New Class</a>
```

Comment:
Was thinking of having a quick route to dashboard#create for this route, as opposed to just dashboard because from the dashboard they will have to scroll down, and I want to minimize actions from the user as much as possible.

Reply:
Yeah for some reason the #links don't seem to work.

Reply:
Don't delete this line, will figure out and get the anchor links to work